src/System.Numerics.Vectors/ref/System.Numerics.Vectors.csproj

```diff
@@ -16,12 +16,9 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard1.0-Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Release|AnyCPU'" />
-  <ItemGroup Condition="'$(TargetGroup)' != 'netstandard'">
```

Comment:
Ok, then just move `netstandard` from `PackageConfigurations` to `BuildConfigurations` in the ref's props.  We might also want to add a `netstandard` config for the lib like we've done for other packages, so that we have a clean ns.dll based impl as well.  If you don't feel like doing that here lets open an issue on it.